include/can/core/ids.hpp

```diff
@@ -87,6 +87,8 @@ enum class MessageId {
     sensor_diagnostic_response = 0x89,
     bind_sensor_output_request = 0x8a,
     bind_sensor_output_response = 0x8b,
+    peripheral_info_request = 0x8c,
```

Comment:
Was this changed by hand or through the code gen?

Reply:
by hand - I branched off of the wrong branch, and I've got to cherry pick that